src/cli/config.rs

```diff
@@ -48,6 +48,18 @@ fn find_config_file(mut directory: PathBuf, recursive: bool) -> Result<Option<Co
     }
 }
 
+pub fn find_ignore_file_path(mut directory: PathBuf, recursive: bool) -> Result<Option<PathBuf>> {
```

Comment:
Doesn't look like this function can ever error, so it can probably just return `Option<PathBuf>` and remove the errors

Reply:
Addressed.